static/index.html

```diff
@@ -35,22 +35,22 @@ <h1 class="title">Remote Play</h1>
       </div>
     </header>
     <main class="container">
-      <button onclick="callEndpoint('/toggle')" class="button full-width">
+      <button onclick="callEndpoint('/toggle')"  id="toggle" class="button full-width">
         Play/Pause
       </button>
-      <button onclick="callEndpoint('/mute')" class="button full-width">
+      <button onclick="callEndpoint('/mute')" id="mute"  class="button full-width">
         Mute
       </button>
-      <button onclick="callEndpoint('/volume_down')" class="button volume_down">
+      <button onclick="callEndpoint('/volume_down')" id="volume_down"  class="button volume_down">
         Volume -
       </button>
-      <button onclick="callEndpoint('/volume_up')" class="button volume_up">
+      <button onclick="callEndpoint('/volume_up')" id="volume_up" class="button volume_up">
         Volume +
       </button>
-      <button onclick="callEndpoint('/seek_left')" class="button back">
+      <button onclick="callEndpoint('/seek_left')" id="seek_left" class="button back">
         &lt;&lt;
       </button>
-      <button onclick="callEndpoint('/seek_right')" class="button forward">
+      <button onclick="callEndpoint('/seek_right')" id="seek_right" class="button forward">
```

Comment:
We don't need this if we change our approach (per my other comment), so probably we can revert this.

Reply:
indeed :D